content/appleboy/drone-sftp-cache/index.md

```diff
@@ -0,0 +1,101 @@
+---
+date: 2017-04-11T00:00:00+00:00
+title: SFTP Cache
+author: appleboy
+tags: [ publish, ssh, sftp, cache ]
+repo: appleboy/drone-sftp-cache
+logo: sftp_cache.svg
+image: appleboy/drone-sftp-cache
+---
+
+Use this plugin for caching build artifacts to speed up your build times. This plugin can create and restore caches of any folders.
+
+```yaml
+pipeline:
+  restore_cache:
+    image: applebot/drone-sftp-cache
+    server: example.com
+    port: 22
+    username: test
+    password: test
+    path: /var/cache/drone
+    restore: true
+    mount:
+      - node_modules
+
+  build:
+    image: node:latest
+    commands:
+      - npm install
+
+  rebuild_cache:
+    image: applebot/drone-sftp-cache
+    server: example.com
+    port: 22
+    username: test
+    password: test
+    path: /var/cache/drone
+    rebuild: true
+    mount:
+      - node_modules
+```
+
+Example configuration for login with user private key:
+
+```diff
+pipeline:
+  rebuild_cache:
+    image: applebot/drone-sftp-cache
+    server: example.com
+    port: 22
+    username: test
+-   password: test
++   key: plain_text_of_user_private_key
+    path: /var/cache/drone
+    rebuild: true
+    mount:
+      - node_modules
+```
+
+Example configuration for ignoring creates a hash file name based on branch name:
```

Comment:
https://github.com/drone/drone-plugin-index/pull/40/commits/c08d5f443d5ea3446bae6bcefa3ea1eebe60c7e3

see the detail https://github.com/appleboy/drone-sftp-cache/blob/master/plugin.go#L99-L103